Documentation/Debugger/natvis/Natvis.md

```diff
@@ -0,0 +1,11 @@
+# Natvis - Custom views for native objects
+
+Natvis framework is a framework that allows developers to write custom schemas to help visualize native objects. A subset of this framework has been ported to the C/C++ extension, residing in the [MIEngine](https://github.com/Microsoft/MIEngine) shared component. If additional features that are not implemented are requested, please file an [issue](https://github.com/Microsoft/MIEngine/issues) on the MIEngine GitHub page with details of what is missing.

```

Comment:
Since the C++ extension supports both vsdbg and MIEngine scenarios, it might be better to clarify this.

Reply:
@gregg-miskelly is that better worded?